template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProviderTest.java

```diff
@@ -45,6 +46,7 @@ public void testGetRangerAwsCloudStorageServiceConfigs() {
                 .withBlueprintView(mock(BlueprintView.class))
                 .withCloudPlatform(CloudPlatform.AWS)
                 .withProductDetails(new ClouderaManagerRepo().withVersion("7.2.2"), List.of())
+                .withStackType(StackType.DATALAKE)
```

Comment:
What about datahub (workload) ?

Reply:
This config is applicable for both datalake as well as datahub for AWS and Azure so I added datalake to make it work. I will add more tests for datahub as well.